packages/edit-site/src/components/navigate-to-link/index.js

```diff
@@ -23,15 +38,16 @@ export default function NavigateToLink( { url, activeId, onActiveIdChange } ) {
 				if ( success ) {
 					let newTemplateId = data.ID;
 					if ( newTemplateId === null ) {
-						const { getEntityRecords } = select( 'core' );
-						newTemplateId = getEntityRecords(
-							'postType',
-							'wp_template',
-							{
-								resolved: true,
-								slug: data.post_name,
-							}
-						)[ 0 ].id;
+						newTemplateId = (
```

Comment:
is this updating the template which would be used to render the current page? just curious to understand this bit of the code.

Reply:
It finds the correct template for the new page, and loads it in the editor.